src/rules.js

```diff
@@ -0,0 +1,28 @@
+const validGuesses = "abcdefghijklmnopqrstuvwxyz".split("");
+export class Rules {
+
+  constructor(secret_word) {
+    this.secret_word = secret_word
+  }
+
+  isGood(guess) {
+    if (this.isCorrectlyGuessed(guess) &&
+         this.isValid(guess)) {
+      return true
+    } else {
+      return false
+    }
+  }
+
+  isCorrectlyGuessed(guess) {
+    return this.secret_word.includes(guess);
+  }
+
+  isValid(guess) {
+    return validGuesses.includes(guess);
+  }
+
+  getWord() {
```

Comment:
Shouldn't this be called something like `sercretWord` as it does return that?
I never implemented Hangman, but it `Rules` the right place to store data as a secret word?